server/src/main/java/org/elasticsearch/index/mapper/vectors/SparseVectorFieldMapper.java

```diff
@@ -112,6 +107,15 @@ public Query termQuery(Object value, SearchExecutionContext context) {
             return FeatureField.newLinearQuery(name(), indexedValueForSearch(value), DEFAULT_BOOST);
         }
 
+        @Override
+        public Query existsQuery(SearchExecutionContext context) {
+            // No support for exists queries prior to this version
+            if (context.getIndexSettings().getIndexVersionCreated().before(NEW_SPARSE_VECTOR_INDEX_VERSION)) {
+                throw new IllegalArgumentException("[sparse_vector] fields do not support [exists] queries");
```

Comment:
I **think** (not completely sure thought), we need to introduce a new index version, because we are changing  the index data (adding a new data to `_field_names`.

Reply:
@mayya-sharipova it does

Reply:
Ah fair enough! I was under the impression that it'd need a new index version if the changes didn't make it into the same release (i.e. waiting until Oct 3).

Reply:
Will update accordingly :)